lms/djangoapps/django_comment_client/forum/views.py

```diff
@@ -287,7 +287,9 @@ def forum_form_discussion(request, course_key):
             'is_course_cohorted': is_course_cohorted(course_key),  # still needed to render _thread_list_template
             'sort_preference': user.default_sort_key,
             'category_map': course_settings["category_map"],
-            'course_settings': json.dumps(course_settings)
+            'course_settings': json.dumps(course_settings),
+            'disable_courseware_js': True,
```

Comment:
The "disable_courseware_js" makes me nervous. I can't pin down exactly why, but I wanted to share my unease. :(